features/step_definitions/basic_steps.rb

```diff
@@ -90,6 +90,8 @@ def goto(page)
     user_registration_path
   when 'create restaurant'
     new_restaurant_path
+  when 'dishes'
+    dishes_path
```

Comment:
It could be used in Line 21 of features/dish_category.feature ? 
But if it was it should have given an error of 'no route matches'